src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/jobservices/ResetNullEventTimestampMessageJobTest.java

```diff
@@ -75,23 +87,60 @@ void should_return_empty_response_for_empty_message_ids() {
                        .run().isEmpty());
     }
 
+    @Test
+    void should_be_equal_when_event_timestamp_is_set() throws JsonProcessingException {
+        EventInformation eventMessageFromEntity = mock(EventInformation.class);
+        List<CaseEventMessageEntity> nullEventTimestampEntityList = new ArrayList<>();
+
+        caseEventMessageEntityMap.put("messageId", "messageId_5");
+        caseEventMessageEntityMap.put("eventTimeStamp", "2022-12-08T15:16:41.805481");
+        CaseEventMessageEntity nullEventTimestampEntity = caseEventMessageEntity
+            .buildMessageEntity(caseEventMessageEntityMap, MessageState.UNPROCESSABLE);
+
+        nullEventTimestampEntityList.add(nullEventTimestampEntity);
+        when(caseEventMessageRepository.findByMessageId(messageIds)).thenReturn(nullEventTimestampEntityList);
+
+        for (int timestamps = 0; timestamps < nullEventTimestampEntityList.size(); timestamps++) {
+            CaseEventMessageEntity messages = nullEventTimestampEntityList.get(timestamps);
+            when(objectMapper.readValue(messages.getMessageContent(), EventInformation.class))
+                .thenReturn(eventMessageFromEntity);
+            messages.setEventTimestamp(eventMessageFromEntity.getEventTimeStamp());
+            assertEquals(eventMessageFromEntity.getEventTimeStamp(), nullEventTimestampEntity.getEventTimestamp());
+        }
+
+        resetNullEventTimestampProblemMessageJob.run();
```

Comment:
Not sure what this test is attempting to do?  Is the system under test the `run()` method?  In which case how is it being tested here?  You're just setting the timestamp in the test above already.

Reply:
This test was meant to try and add more coverage to `ResetNullEventTimestampMessageJobTest` but it doesn't kill the mutation as with some of the other tests i've written locally, i've removed it as the tests seem ok the way they are for now.